axelrod/strategies/titfortat.py

```diff
@@ -750,3 +716,59 @@ def strategy(self, opponent: Player) -> Action:
     def reset(self):
         super().reset()
         self.is_defector = False
+
+
+class NTitsForMTats(Player):
+    """
+    A parameterizable Tit-for-Tat,
+    The arguments are :
+    1) the number of defection before retaliation
+    2) the number of retaliations
+
+    Names:
+
+    - N Tit(s) For M Tat(s): Original name by Marc Harper
+    """
+
+    name = 'N Tit(s) For M Tat(s)'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': False,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def __init__(self, N: int=1, M: int=1) -> None:
+        """
+        Parameters
+        ----------
+        N, int
```

Comment:
Because of the type hints let's drop the type in the docstrings (in case at some point they are changed and we forget to change them in the docstrings).